### DISCUSSION TITLE
Filters on the VSCode Outline panel for Apex classes

### DISCUSSION THREAD

**Discussion by ImJohnMDaniel** [forcedotcom/salesforcedx-vscode#5895] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I have an Apex class file open in the IDE, and I view the Outline view panel, the results are not filterable

**Describe the solution you'd like**
It would be nice to be able to filter the results
* "See only class variables"
* "See only public methods"
* "See only private methods"
* "See only inner classes"

**Describe alternatives you've considered**
There really is no alternative to this idea other than straight searching through the file until you find what you are looking for.

[a comment by an automated account is not shown]

**Comment by peternhale** [comment]
@ImJohnMDaniel thanks for the suggestion. I am going to move your feature request into the discussion section of this repo.
